src/osx/core/bitmap.cpp

```diff
@@ -62,19 +62,33 @@ class WXDLLEXPORT wxBitmapRefData: public wxGDIRefData
     void Free();
     void SetOk( bool isOk) { m_ok = isOk; }
 
+#if wxOSX_BITMAP_NATIVE_ACCESS
+    int GetWidth() const { return CGBitmapContextGetWidth(m_hBitmap); }
+    int GetHeight() const { return CGBitmapContextGetHeight(m_hBitmap); }
+    int GetDepth() const { return CGBitmapContextGetBitsPerPixel(m_hBitmap); }
+    int GetBytesPerRow() const { return CGBitmapContextGetBytesPerRow(m_hBitmap); }
+    bool HasAlpha() const {
+        CGImageAlphaInfo alpha = CGBitmapContextGetAlphaInfo(m_hBitmap);
+        return !( alpha == kCGImageAlphaNone || alpha == kCGImageAlphaNoneSkipFirst || alpha == kCGImageAlphaNoneSkipLast) ;
```

Comment:
It would be nice to avoid overlong (>> 80 characters) lines, they make side-by-side diffs less readable.

Reply:
Thanks, done